### ISSUE TITLE
Registered events

### ISSUE THREAD

**Issue by mathaywarduk** [dx-shu/events#88]
Hover state is on XD protoype. Might have to switch to comment mode to view it.

However...

White on the teal fails accessibility. So instead of teal use: #B6004C

**Comment by nternetinspired** [comment]
@mathaywarduk not #ba0046 which is already in the pallette? I've not seen #ba004c used anywhere else…

**Comment by mathaywarduk** [comment]
@nternetinspired #ba0046 would work yep
